### PR TITLE
Update astraDB Application Quickstart Python README.md

### PR DESCRIPTION
**Pull request by beajohnson**

### CODE REVIEW

File: astraDBApplicationQuickstartPython/README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 # Overview
 Learn how to connect your app to Astra using Python.
 
-**In this guide, we'll learn how to**
+**In this guide, you'll learn how to**
 - Set up our Python development environment
```
Comment (SonicDMG) [inline astraDBApplicationQuickstartPython/README.md:5]: Should this be "Set up your Python development environment"?

File: astraDBApplicationQuickstartPython/README.md [hunk 2]
```diff
@@ -121,9 +121,9 @@ auth_provider = PlainTextAuthProvider(username, token)
 cluster = Cluster(cloud=cloud_config, auth_provider=auth_provider)
 ```
 
-Here, we're using our **secureBundleLocation** to set up our cloud configuration.  We're also initializing the **PlainTextAuthProvider**, and using both of these to configure our cluster object.
+Here, use you **secureBundleLocation** to set up your cloud configuration.  You're also initializing the **PlainTextAuthProvider**, and using both of these to configure our cluster object.
```
Comment (SonicDMG) [inline astraDBApplicationQuickstartPython/README.md:124]: *your

**Review by SonicDMG — CHANGES_REQUESTED** [review]

**Comment by gstachni** [comment]
@beajohnson Can we also make a change to the config.json to update the title of the guide from "How to get started with your Python application and Astra DB 🐍" to "Get started with your Python application and Astra DB 🐍"?

File: astraDBApplicationQuickstartPython/README.md (hunk 1 above)
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:5]:
```suggestion
- Set up your Python development environment
```

Comment (beajohnson) [reply astraDBApplicationQuickstartPython/README.md:5]: Yes

File: astraDBApplicationQuickstartPython/README.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 
-**In this guide, we'll learn how to**
+**In this guide, you'll learn how to**
 - Set up our Python development environment
 - Connect to Astra DB and execute a static query
 - Execute a query using a prepared statement
 
 # Prerequisites
-- You also should have a recent version of Python 3 installed.
+- You also should have a recent version of Python 3.
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:10]:
```suggestion
You must have installed a recent version of Python 3.
```

File: astraDBApplicationQuickstartPython/README.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 
 ## 1 Create a database
-If you haven't already, go ahead and create a new Astra DB database. 
+If you haven't already done so, create a new Astra DB database.
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:13]:
```suggestion
If you haven't already done so, create a new {astra_db} database. 
```

File: astraDBApplicationQuickstartPython/README.md [hunk 5]
```diff
@@ -91,17 +91,17 @@ env | grep ASTRA
 ```
 
 ## 4 Create a new project and connect to Astra DB
-Let's start by creating a new Python program to connect to Astra DB.  Open your favorite IDE and create a new file called **testAstraDB.py**.  We will start by adding three imports:
+Start by creating a new Python program to connect to Astra DB.  Open your favorite IDE and create a new file called **testAstraDB.py**. Add the following three imports:
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:94]:
```suggestion
Create a new Python program to connect to {astra_db}.  Open your favorite IDE and create a new file called **testAstraDB.py**. Add the following three imports:
```

File: astraDBApplicationQuickstartPython/README.md [hunk 6]
```diff
@@ -91,17 +91,17 @@ env | grep ASTRA
...
 
 ```python
 from cassandra.cluster import Cluster
 from cassandra.auth import PlainTextAuthProvider
 import os
 ```
 
-This makes it possible for you to establish a secure connection (**PlainTextAuthProvider**) to your Astra DB cluster (**Cluster**).  We'll also need **os** to access the environment variables.
+This makes it possible for you to establish a secure connection (**PlainTextAuthProvider**) to your Astra DB cluster (**Cluster**).  You'll also need **os** to access the environment variables.
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:102]:
```suggestion
This makes it possible for you to establish a secure connection (**PlainTextAuthProvider**) to your {astra_db} cluster (**Cluster**).  You'll also need **os** to access the environment variables.
```

File: astraDBApplicationQuickstartPython/README.md [hunk 7]
```diff
@@ -142,30 +142,30 @@ cluster name = cndb
...
-For our use case, we'll read in a keyspace and table name, and then display the columns present in that table.  Let's create variables in our code to read those arguments from the command line:
+For this use case, read in a keyspace and table name, and then display the columns present in that table. Create variables in your code to read those arguments from the command line:
 
 ```python
 keyspace = sys.argv[1]
 table = sys.argv[2]
 ```
 
-Next, we'll create our query string and prepare it.  Preparing our query string adds it to the prepared statement cache of our Astra DB cluster, preventing it from needing to be parsed before each use.
+Next, create your query string and prepare it. Preparing your query string adds it to the prepared statement cache of your Astra DB cluster, preventing the need to parse it before each use.
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:160]:
```suggestion
Next, create your query string and prepare it. Preparing your query string adds it to the prepared statement cache of your {astra_db} cluster, preventing the need to parse it before each use.
```

File: astraDBApplicationQuickstartPython/README.md [hunk 8]
```diff
@@ -142,30 +142,30 @@ cluster name = cndb
...
 
 ```python
 pStatement = session.prepare("""
     SELECT column_name FROM system_schema.columns WHERE keyspace_name=? AND table_name=?;
 """)
 ```
 
-Now, we'll execute the prepared statement while passing the **keyspace** and **table** variables.  Then, we will process and display the results.
+Now, execute the prepared statement while passing the **keyspace** and **table** variables.  Then, process and display the results.
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:168]:
```suggestion
Execute the prepared statement while passing the **keyspace** and **table** variables. Then, process and display the results.
```

File: astraDBApplicationQuickstartPython/README.md [hunk 9]
```diff
@@ -182,7 +182,7 @@ When you run the code, be sure to pass the names of an existing **keyspace** and
 python testAstraDB.py system local
 ```
 
-_For this example, we're just using "**system**" (keyspace) and "**local**" (table name) to ensure you get some output. You could use whatever keyspace and table names you have access to in the database you are using._
+_For this example, just use "**system**" (keyspace) and "**local**" (table name) to ensure you get some output. You could use whatever keyspace and table names you have access to in the database you are using._
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:185]:
```suggestion
_For this example, use "**system**" (keyspace) and "**local**" (table name) to ensure you get some output. You could use whatever keyspace and table names you have access to in the database you are using._
```

File: astraDBApplicationQuickstartPython/README.md [hunk 10]
```diff
@@ -201,7 +201,7 @@ host_id
 ```
 
 ## 6 Summary
-In summary, you learned to configure your local Python development environment. We've learned to use environment variables for referencing our Astra DB credentials and secure connect bundle. Finally, we've learned how to run a simple, static query – including how to query results with a prepared statement. This should provide a solid foundation for building more complex Python applications.
+In summary, you learned to configure your local Python development environment. We've learned to use environment variables for referencing our Astra DB credentials and secure connect bundle. Finally, you've learned how to run a simple, static query – including how to query results with a prepared statement. This should provide a solid foundation for building more complex Python applications.
 
 Note, that the complete code for [testAstraDB.py](https://github.com/riptano/astra-portal-getting-started/blob/main/astraDBApplicationQuickstartPython/testAstraDB.py) can be found in this repo.
```
Comment (beajohnson) [inline astraDBApplicationQuickstartPython/README.md:206]:
```suggestion
**NOTE:** 
The complete code for [testAstraDB.py](https://github.com/riptano/astra-portal-getting-started/blob/main/astraDBApplicationQuickstartPython/testAstraDB.py) can be found in this repo.
```

**Pull request merged**